hera_sim/defaults.py

```diff
@@ -31,30 +31,30 @@ class Defaults:
     Examples
     --------
     To set the default parameters to those appropriate for the H2C
-    observing season (and activate the use of those defaults):
+    observing season (and activate the use of those defaults)::
 
-    hera_sim.defaults.set('h2c')
+        hera_sim.defaults.set('h2c')
 
     To set the defaults to a custom set of defaults, you must first
     create a configuration YAML. Assuming the path to the YAML is
     stored in the variable `config_path`, these defaults would be set
-    via the following line:
+    via the following line::
 
-    hera_sim.defaults.set(config_path)
+        hera_sim.defaults.set(config_path)
 
-    To revert back to using defaults defined in function signatures:
+    To revert back to using defaults defined in function signatures::
 
-    hera_sim.defaults.deactivate()
+        hera_sim.defaults.deactivate()
 
-    To view what the default value is for a particular parameter, do:
+    To view what the default value is for a particular parameter, do::
 
-    hera_sim.defaults(parameter),
+        hera_sim.defaults(parameter),
 
     where `parameter` is a string with the name of the parameter as
     listed in the configuration file. To view the entire set of default
-    parameters, use:
+    parameters, use::
 
-    hera_sim.defaults()
+        hera_sim.defaults()
```

Comment:
To follow-up on this, there are some examples (I think in the `sigchain` module?) where only a single colon precedes the actual example, rather than the sets of two colons used here.

Reply:
Yeah, RST requires a double-colon before code blocks (and they should be indented as they are here). I'll go find other examples to fix them too :-)